internal/rbd/nodeserver.go

```diff
@@ -1127,6 +1127,8 @@ func (ns *NodeServer) NodeExpandVolume(
 	imgInfo, err := lookupRBDImageMetadataStash(volumePath)
 	if err != nil {
 		log.ErrorLog(ctx, "failed to find image metadata: %v", err)
+
+		return nil, status.Error(codes.Internal, err.Error())
```

Comment:
It would be nice to have a little more context around `err.Error()`. If the error gets returned (and logged somewhere), having details about the failure can be helpful.

Reply:
`lookupRBDImageMetadataStash` has recorded detailed error context and log will print it.